packages/parser/lib/interfaces/IASTNode.ts

```diff
@@ -0,0 +1,6 @@
+import EvalExprTypeEnum from "../../../core/lib/enums/EvalExprTypeEnum";
```

Comment:
we can't import cross package like this

Reply:
yup, I'm going to change this when is fully ts